3-limit-service-time/main.go

```diff
@@ -21,10 +25,19 @@ type User struct {
 // HandleRequest runs the processes requested by users. Returns false
 // if process had to be killed
 func HandleRequest(process func(), u *User) bool {
+	if u.TimeUsed >= 10 {
```

Comment:
here you track 10 second per user accumulated, so I assume you're going for an advanced solution - but you do not have `if condition` to allow premium users to go over  10s limit